params/index.yaml

```diff
@@ -27,3 +27,18 @@ BlockRoot:
   description: |
     Block root.
     \<hex encoded blockRoot with 0x prefix\>.
+SkipRandaoVerification:
+  name: skip_randao_verification
+  in: query
+  required: false
+  description: |
+    Skip verification of the `randao_reveal` value. If this flag is set then the
+    `randao_reveal` must be set to the point at infinity (`0xc0..00`). 
+    For backward compatibility, this query parameter can be set empty, but recommended to set a correct value when calling. 
+  schema:
+    oneOf: 
+      - type: boolean
+      - type: string
+        minLength: 0
+        maxLength: 0
```

Comment:
Maybe just remove the boolean type and only specify `type: string` with the min/max length. This makes it clear from an openapi spec point of view on what a valid value should look like.

Reply:
Yeah I think this is a better idea.